SteamList/Models/GameModel.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-struct Game {
+class Game {
```

Comment:
Why have you decided to change struct to class?

Reply:
Since I moved the `adjustIsFavorite()` method to cells class, I need to pass the `Game` object to cell class and modify it there,  so I thought the best way to do this is to pass the Game object by reference but struct would not let me do that.